docs/column.pin.md

```diff
@@ -1,12 +1,25 @@
 # qgrid 
 ## markup
 ```html
-<q-grid rows='data'></q-grid>
+<q-grid rows="$ctrl.rows">
+	<q-grid:columns>
+		<q-grid:column pin="left" key="name.last" title="Last Name" value="$ctrl.getLastName($row)"></q-grid:column>
+		<q-grid:column key="birthday" title="Birthday"></q-grid:column>
+		<q-grid:column pin="right" key="gender" title="Gender"></q-grid:column>
+	</q-grid:columns>
+</q-grid>
 ```
 ## script
 ```javascript
-controller.$inject = ['$scope'];
-function controller($scope){
-	$scope.data = [];
+controller.$inject = ['$scope', 'qgrid'];
+function controller($scope, qgrid){
+  $scope.columns = [
+          {key: 'firstName', label: 'First Name', type: 'text', isVisible: true, isDefault: true, isPinned: 'left'},
+          {key: 'lastName', label: 'Last Name', type: 'text', isVisible: true, isDefault: true, isPinned: 'left'},
+          {key: 'birthDate', label: 'Date of Birth', type: 'date', isVisible: true, isDefault: true},
+          {key: 'location', label: 'Location', type: 'date', isVisible: true, isDefault: true, isPinned: 'right'},
+          {key: 'zipCode', label: 'Zip', type: 'number', isVisible: false, isDefault: false, isPinned: 'right'}
+      ];
+   
```

Comment:
pinned it sounds like a boolean state for me :)
however we need to have common style pin vs pinned, canResize vs resizable, canMove vs moveable, isVisible vs visible, isDefault vs 'default' etc.